src/components/decvotesbtn.jsx

```diff
@@ -0,0 +1,47 @@
+import axios from 'axios'
+import {BiDownvote} from 'react-icons/bi'
+
+export default function DecVotesBtn({reviewID, setSingleReview, setReviewList, setDisableInc, disableDec, setDisableDec, setReviewByCategory}){
+    function incVotes(){
```

Comment:
I appreciate that you are trying to make these buttons re-usable across the 3 components that you have made, but I think dec votes and inc votes components could be refactored into a single component.

That being said that this is a really big and complex function and I think it could be simplified significantly, rather than passing all the setState functions into the components you should be able to pass a single change votes function in there, and create the individual implementation of that function in all those places